bgpd/bgp_vty.c

```diff
@@ -14640,7 +14644,7 @@ static void bgp_show_peer(struct vty *vty, struct peer *p, bool use_json,
 	}
 
 	/* Remote address. */
-	if (p->su_remote) {
+	if (p->su_remote && (p->status == Established)) {
```

Comment:
We can't just change the display of this data to nothing.  People have scripts and there is a process to changing the cli that we must go through

Reply:
ditto